test/testdata/lsp/fast_path/static_field_change_type__def.1.rbupdate

```diff
@@ -0,0 +1,5 @@
+# typed: strict
+# assert-fast-path: static_field_change_type__def.rb,static_field_change_type__use.rb
+
+STATIC_FIELD = T.let({}, T::Hash[Symbol, ]) # error-with-dupes: Not enough arguments provided
+                               # ^^^^^^ error-with-dupes: Wrong number of type parameters
```

Comment:
I have a slight preference for breaking the `error-with-dupes` errors so that there's one `# ^^^ error` comment per dup each on its own line.

Separately, it's annoying that we seem to report so many errors here, even on the slow path:

https://github.com/sorbet/sorbet/issues/5822